### ISSUE TITLE
ASCII maps for cartesian always behave as though they are reading for quarter-symmetry

### ISSUE THREAD

**Issue by youngmit** [terrapower/armi#332]
Consider this grid blueprint from the C5G7 example:
```
    UO2 grid:
        symmetry: full
        geom: cartesian
        lattice pitch:
            x: 1.26
            y: 1.26
        lattice map: |
            U  U  U  U  U  U  U  U  U  U  U  U  U  U  U  U  U
            U  U  U  U  U  U  U  U  U  U  U  U  U  U  U  U  U
            U  U  U  U  U  GT U  U  GT U  U  GT U  U  U  U  U
            U  U  U  GT U  U  U  U  U  U  U  U  U  GT U  U  U
            U  U  U  U  U  U  U  U  U  U  U  U  U  U  U  U  U
            U  U  GT U  U  GT U  U  GT U  U  GT U  U  GT U  U
            U  U  U  U  U  U  U  U  U  U  U  U  U  U  U  U  U
            U  U  U  U  U  U  U  U  U  U  U  U  U  U  U  U  U
            U  U  GT U  U  GT U  U  FC U  U  GT U  U  GT U  U
            U  U  U  U  U  U  U  U  U  U  U  U  U  U  U  U  U
            U  U  U  U  U  U  U  U  U  U  U  U  U  U  U  U  U
            U  U  GT U  U  GT U  U  GT U  U  GT U  U  GT U  U
            U  U  U  U  U  U  U  U  U  U  U  U  U  U  U  U  U
            U  U  U  GT U  U  U  U  U  U  U  U  U  GT U  U  U
            U  U  U  U  U  GT U  U  GT U  U  GT U  U  U  U  U
            U  U  U  U  U  U  U  U  U  U  U  U  U  U  U  U  U
            U  U  U  U  U  U  U  U  U  U  U  U  U  U  U  U  U
```

When the contents of `lattice map` are read in, the `asciimaps` code is treating the bottom-left `U` as (0,0), and placing all of the objects in the first quadrant. This is what you would expect in a quarter-symmetric case. However in a full-symmetry case, one would expect the (0,0) to lie on the `FC` in the center. We need to do a symmetry-aware offset when reading `lattice maps:`

**Comment by jakehader** [comment]
@youngmit, we saw this in a benchmark model of ZPPR as well within the grid editor.

**Comment by jakehader** [comment]
Closing since this was fixed with #346.
